pdfminer/pdftypes.py

```diff
@@ -185,6 +186,23 @@ def stream_value(x):
     return x
 
 
+def decompress_corrupted(data):
```

Comment:
Could you add a docstring saying that this extracts the compressed file as much as possible?